src/flatten.rs

```diff
@@ -445,20 +448,36 @@ impl Flattener {
                             .unwrap_or(content.len());
 
                         let name = &content[name_start..name_end];
+                        trace!("found name {name}");
+
+                        let mut new_name = None;
 
                         if let Some(ast_id) = exported_symbols.get(name) {
-                            let new_name = top_level_names.get(ast_id).unwrap();
-                            updates.entry(path.to_path_buf()).or_default().insert((
-                                src_start + name_start,
-                                src_start + name_end,
-                                new_name.to_string(),
-                            ));
-                        } else {
-                            updates.entry(path.to_path_buf()).or_default().insert((
-                                src_start + tag_start,
-                                src_start + name_end,
-                                "".to_string(),
-                            ));
+                            if let Some(name) = top_level_names.get(ast_id) {
+                                new_name = Some(name);
+                            } else {
+                                trace!("ast id {ast_id} cannot be matched to top-level identifier");
+                                trace!("known top-level identifiers: {:?}", top_level_names);
```

Comment:
these can be combined
```suggestion
                                trace!(identifiers=?top_level_names, "ast id {ast_id} cannot be matched to top-level identifier");
```